backend/src/v5/services/eventsListener/components/modelEvents.js

```diff
@@ -88,7 +88,7 @@ const revisionAdded = async ({ teamspace, project, model, revId, modelType, cali
 			{ _id: 0, tag: 1, author: 1, timestamp: 1, desc: 1, rFile: 1, format: 1, statusCode: 1, revCode: 1 });
```

Comment:
I recall talking to @Amantini1997  about this so maybe it's fixed in a different milestone, but the drawing panel should be visible if the enters the viewer with a drawing ID in the query. I.e. it should look like this:

![image](https://github.com/user-attachments/assets/78538a1b-a799-4ed8-a193-85b483740775)

At the moment it looks like this:
![image](https://github.com/user-attachments/assets/1629664b-d03d-4af8-80e7-f93085205036)